src/main/js/components/dropdowns/utils.js

```diff
@@ -19,15 +19,26 @@ function generateDropdown(element, callback, immediate) {
   tippy(
     element,
     Object.assign({}, Templates.dropdown(), {
-      hideOnClick: element.dataset["hideOnClick"] !== "false",
+      hideOnClick: false,
       onCreate(instance) {
         const onload = () => {
           if (instance.loaded) {
             return;
           }
 
-          instance.popper.addEventListener("click", () => {
-            instance.hide();
+          document.addEventListener("click", (event) => {
+            const visibleDropdowns = Array.from(document.querySelectorAll('[data-tippy-root]'))
+              .filter(dropdown => window.getComputedStyle(dropdown).visibility === 'visible');
+
+            const isClickInAnyDropdown = visibleDropdowns.some(dropdown =>
```

Comment:
Why check for click on visible dropdowns and not all of them? I think something like|

`isClickInAnyDropdown = !!event.target().closest('[data-tippy-root]')`

should be enough (?)

Reply:
I have updated the code, and it works great. Thank you.